pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -64,7 +68,7 @@ for arg; do
             --builder) builder=true;;
             --help) usage; exit;;
             *)
-                argi=$(($argi + 1))
+                argi=$((argi + 1))
```

Comment:
since this is explicitly using bash you could replace this line with `((argi++))`